cloud/ovirt/ovirt_vms.py

```diff
@@ -191,6 +191,20 @@
             - "C(nic_gateway) - If boot protocol is static, set this gateway to network interface of Virtual Machine."
             - "C(nic_name) - Set name to network interface of Virtual Machine."
             - "C(nic_on_boot) - If I(True) network interface will be set to start on boot."
+    cloud_init_nics:
+        description:
+            - "List of dictionaries representing network interafaces to be setup by cloud init."
+            - "This option is used, when user need to setup more network interfaces via cloud init."
+            - "If one network interface is enough, user should use C(cloud_init) I(nic_*) parameters. C(cloud_init) I(nic_*) parameters
+               are merged with C(cloud_init_nics) parameters."
+            - "Dictionary can contain following values:"
+            - "C(nic_boot_protocol) - Set boot protocol of the network interface of Virtual Machine. Can be one of None, DHCP or Static."
```

Comment:
Maybe it's better to display value in lowercase as values are not case sensitive: none, dhcp, static